src/main/kotlin/com/msd/robot/domain/Robot.kt

```diff
@@ -135,35 +165,75 @@ class Robot(
             field = value
         }
 
+    /**
+     * Moves this `Robot` to a given `Planet`, unless the `Planet` is currently blocked.
```

Comment:
The move is only blocked when this Robot's current Planet is blocked. Please correct this method description.